### PR TITLE
Step 3 Week3

### PR DESCRIPTION
**Pull request by zyelcrier**
-add Gson

### CODE REVIEW

**Comment by zyelcrier** [comment]
Last commit is most recent the above commit with the same name was unintentionally unfinished
-last commit includes all finalized changes 
- two Servelets

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,6 +19,8 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```
Comment (ljfranklin) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: Doesn't look like anything really changed in this file, remove the unused imports and extra newline at the bottom.

File: portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import java.util.*;
+import com.google.gson.Gson ;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/message")
+public class MessageServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> messages= new ArrayList<>();
```
Comment (ljfranklin) [inline portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java:31]: Run a formatter on this file, e.g. needs space before `=`

File: walkthroughs/week-3-server/examples/server-stats/src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (ljfranklin) [inline walkthroughs/week-3-server/examples/server-stats/src/main/webapp/WEB-INF/appengine-web.xml:1]: Why was this file deleted?

Comment (zyelcrier) [reply walkthroughs/week-3-server/examples/server-stats/src/main/webapp/WEB-INF/appengine-web.xml:1]: that was an accident and im not quite sure how to fix it @ljfranklin

Comment (ljfranklin) [reply walkthroughs/week-3-server/examples/server-stats/src/main/webapp/WEB-INF/appengine-web.xml:1]: @zyelcrier try `git checkout master YOUR_FILE`. That tell git to update YOUR_FILE to match what it looks like on the master branch.

File: portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java (hunk 2 above)
Comment (ljfranklin) [reply portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java:31]: @zyelcrier still need a space before `=`

Comment (zyelcrier) [reply portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java]: can you clarify this for me i dont get what your pointing at  @ljfranklin

Comment (zyelcrier) [reply portfolio/src/main/java/com/google/sps/servlets/MessageServlet.java]: by formatter you mean prettier?
